web/src/message_edit.js

```diff
@@ -42,6 +42,7 @@ const currently_editing_messages = new Map();
 let currently_deleting_messages = [];
 let currently_topic_editing_messages = [];
 const currently_echoing_messages = new Map();
+const uppys = {};
```

Comment:
1. I think you want to do `new Map()` instead of `{}`, to be clear it's not just an empty object.

2. I'd probably call this `upload_objects_by_row`, though maybe that's a bit wordy. Ideally someone looking at the `message_edit` code doesn't need to know exactly what `uppy` is. When we have types, it'll be easier to keep track of what a map does without wordy names.

Reply:
Done, thanks!